tests/test_jwt_functions.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from datetime import datetime, timedelta
+
+import jwt
+
+from app import JWTDecodingError, decode_jwt_token, generate_jwt_token
```

Comment:
this test is a test that should only exists in membrane-backend... which we should depend on here.

Reply:
Removed